helloworld-jersey/src/main/resources/eureka-client.properties

```diff
@@ -47,7 +47,7 @@ eureka.preferSameZone=
 eureka.shouldUseDns=false
 
 # Since shouldUseDns is false, we use the following properties to explicitly specify the route to the eureka servers
-eureka.serviceUrl.default=http://eureka:password@localhost:10011/eureka/
+eureka.serviceUrl.default=https://eureka:password@localhost:10011/eureka
```

Comment:
Connect Eureka without login&password.

Reply:
Hi Jirka, I guess I need to wait for you to merge, because jersey-enabler is dependent on java-enabler. Anyway yesterday I was trying to deploy helloworld-spring but it's not working in https. I'm not sure if you were maybe starting the service with a different configuration, by tomcatRun still use http config.

Reply:
I am not sure if we should change helloworld-spring to HTTPS. I see advantages but also one disadvantage. I was able o double check whether the enabler is not changing the service to https. Enabler should not affect the scheme of the service.